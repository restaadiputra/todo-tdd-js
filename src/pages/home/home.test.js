import React from 'react';
import { render } from '@testing-library/react';
import Home from './home';

test('render without crash', () => {
  const { container } = render(<Home />);
  expect(container.firstChild).toMatchSnapshot();
});
