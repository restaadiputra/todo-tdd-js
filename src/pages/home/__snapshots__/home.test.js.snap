// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without crash 1`] = `
<div
  class="max-w-lg mx-auto mt-4 flex flex-col items-center justify-center"
>
  <img
    alt="todo"
    class="mt-8 mb-12 w-48 md:w-56 lg:w-64"
    src="todo.svg"
  />
  <p
    class="text-gray-800 text-lg text-center px-6"
  >
    Just 3 simple todo app with 3 different method of state management. Use menu above to try each todo.
  </p>
</div>
`;
